src/main/java/io/openshift/booster/catalog/Booster.java

```diff
@@ -27,6 +27,7 @@
    private String buildProfile;
    private String description = "No description available";
    private String boosterDescriptorPath = ".openshiftio/booster.yaml";
+   private boolean disableLaunch = false;
```

Comment:
Perhaps a `supportedDeploymentTypes` or a similar name would be a better fit here?

Reply:
Or perhaps we could use the label feature and avoid introducing another option

Reply:
The supportDeploymentTypes is certainly an option. But I found the names that were used by deplymentType a bit strange (`CD` and `OSIO`).

I don't agree with (ab)using the labels though. Starting with the fact that how would we set them? We would have to implicitly set labels  based on the deployment type (and why only deployment type and not runtime and mission as well?) but that would surely interfere with the labels being used for filtering (all labels need to exist for filtering to work).

Reply:
I agree that `CD` is not a good option name. Perhaps we could change that to something more meaningful (may impact the frontend though).
However labels aren't used anywhere right now (they should exist in the boosters' repository tbh but that's another issue) and afaik filtering happens only if the label is provided. 

A supportedDeploymentTypes in the newly created Selector would be a good choice and then we can read from a label or a new property in the booster YAML if that makes sense

Reply:
Ah there is already the deploymentType in the Selector, maybe that makes sense